fix(tutorials): wordpress & lemp stack MTA-1593 (#269)
scaleway/docs-content: tutorials/installation-lemp-ubuntu-bionic/index.mdx

---
meta:
  title: Installing LEMP-Stack (Linux, Nginx, MySQL, PHP) on Ubuntu Bionic Beaver
  description: How to install LEMP-Stack (Linux, Nginx, MySQL, PHP) on Ubuntu Bionic Beaver (18.04)
content:
  h1: Installing LEMP-Stack (Linux, Nginx, MySQL, PHP) on Ubuntu Bionic Beaver

>   paragraph: How to install LEMP-Stack (Linux, Nginx, MySQL, PHP) on UbuntuBionic Beaver (18.04)

Reviewer: ```suggestion
  paragraph: How to install LEMP-Stack (Linux, Nginx, MySQL, PHP) on Ubuntu Bionic Beaver (18.04)
```

  mysql_secure_installation
  ```

> 2. Enter the current root password for the MariaDB server. when prompted As it not yet set, press `Enter`.

Reviewer 2: ```suggestion
2. Enter the current root password for the MariaDB server. When prompted, as it is not yet set, press `Enter`.
```

> 2. Enter the current root password for the MariaDB server. when prompted As it not yet set, press `Enter`.

Author: ```suggestion
2. Press "Enter" when prompted for the current root password for the MariaDB server, as the password is not yet set.
```

4. Enter the new password and press `Enter`, then repeat this step.

5. Keep the settings proposed by default when prompted by further questions, by pressing `Enter` each time.

## Configuring a Nginx Server-Block

Nginx stores the configuration of virtual hosts in server blocks. All available server blocks are located in the directory `/etc/nginx/sites-available/`.

1. Navigate to the relevant directory:

  ```
  cd /etc/nginx/sites-available/
  ```

2. Create a new server block configuration file and name it after the domain name that points to your Instance. Here we use `example.com`:

  ```
    server {
          listen 80;
          root /var/www/html;

>           index index.php index.html index.htm index.nginx-debian.html;

Reviewer: ```suggestion
          index index.php index.html index.htm;
```
I don't think you need the last file, it refers to the Debian specific index file only

          server_name example.com;

          location / {
                  try_files $uri $uri/ =404;
          }

          location ~ \.php$ {
                  include snippets/fastcgi-php.conf;
                  fastcgi_pass unix:/var/run/php/php7.2-fpm.sock;
          }

          location ~ /\.ht {
                  deny all;
          }
  }
  ```

>   Remember to replace `example.com` and any other information as necessary as per the overview below:

Reviewer 3: ```suggestion
  Remember to replace `example.com` and any other information necessary as per the overview below:
```

  ?>
  ```

> 3. Save the file and point your web browser to `http://example.com/phpinfo.php` (replacing `example.com` with your domain name). When you see an output like the following, PHP is setup correctly:

Reviewer 3: ```suggestion
3. Save the file and point your web browser to `http://example.com/phpinfo.php` (replacing `example.com` with your domain name). When you see an output like the following, it means PHP is set up correctly:
```
